saleor/product/migrations/0186_remove_product_charge_taxes.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.19 on 2023-07-12 10:16
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("product", "0185_unmount_product_charge_taxes"),
+    ]
+
+    database_operations = [
+        migrations.RunSQL(
+            sql="ALTER TABLE product_product DROP COLUMN charge_taxes;",
+            reverse_sql=migrations.RunSQL.noop,
```

Comment:
I know that we don't require `reverse_sql`, but maybe we can add it here for our own convenience in unapllying migrations?
```suggestion
            reverse_sql="""
            ALTER TABLE product_product
            ADD COLUMN charge_taxes
            BOOLEAN;
            """
```